src/Qir/Execution/Tools/Executable/ClangClient.cs

```diff
@@ -1,11 +1,11 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-using System.CommandLine.Invocation;
+using System.Diagnostics;
 using System.Threading.Tasks;
 using Microsoft.Quantum.Qir.Utility;
 
-namespace Microsoft.Quantum.Qir.Executable
+namespace Microsoft.Quantum.Qir.Tools.Executable
 {
     public class ClangClient : IClangClient
```

Comment:
Since this is now going to be distributed as a package, I think most of the classes and interfaces should be changed to `internal`. The only ones that should be left `public` are:
- `IQirDriverGenerator`
- `QirFullStateDriverGenerator`
- `IQirExecutable`
- `QirExecutable`
- `QirFullStateExecutable`